src/cascadia/UIHelpers/TextMenuFlyout.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

#pragma once
```

Probably also worth adding the BODGY comment here and referencing the GH issue for historical reasons.